lib/python/src/bailo/helper/model.py

```diff
@@ -319,6 +339,14 @@ def from_mlflow(self, tracking_uri: str, experiment_id: str):
         if ml_flow:
             client = mlflow.tracking.MlflowClient(tracking_uri=tracking_uri)
             runs = client.search_runs(experiment_id)
+            if len(runs) > 0:
```

Comment:
We can get rid of `>0` It doesn't have any effect on this condition.

Reply:
Done